### DISCUSSION TITLE
急急急，我的货物丢在空洞里了，委托费3000丁尼🤓

### DISCUSSION THREAD

**Discussion by WXZ1025** [share121/inter-knot#2745] [其他]
我是天马速运的外包司机，今天我没注意，货物不小心掉在空洞里了，有没有人帮帮我😭
![联图二维码 (3)](https://github.com/user-attachments/assets/67e7e1cc-63ff-4d64-b00c-38372019ee93)

**Comment by WXZ1025** [comment]
有没有入呀😭😭😭我好急在线等

**Comment by lllliufeng** [comment]
是真的，已经拿到1000[丁尼](https://baike.mihoyo.com/zzz/wiki/content/16/detail?mhy_presentation_style=fullscreen)了

**Comment by CBBC123** [comment]
我经常干这事，这个委托我接了

**Reply by WXZ1025** [reply]
谢谢大佬，办事效率真快😃

**Comment by isle200inf** [comment]
3000菲林😡

**Comment by Shuyluo** [comment]
委托费太少了，有这时间不如去六分街刮点彩票

**Comment by CeceliaHarex** [comment]
不是3000[菲林](https://baike.baidu.com/item/%E8%8F%B2%E6%9E%97/64622252)我不干👊🏻

**Comment by WXZ1025** [comment]
他哥的12000丁尼委托费😡👊，有没有人接？

**Comment by panda114514667** [comment]
他急了，他急了，小心空洞绑架
![Uploading AnimatedEmojies-512px-257.gif…]()
![Uploading AnimatedEmojies-512px-8.gif…]()

**Comment by GrunderIndustries** [comment]
您可以考虑购买我司的MQ-99无人机 然后录入萝卜数据来找回货物
